src/scripts/libs/snippets.js

```diff
@@ -0,0 +1,3 @@
+// from Material Design Lite website
```

Comment:
Happy if you want to re-write, just make it ES2015 please :)

Sorry didn't spot the async / defer script in the HTML. Ignore the module stuff.